helpers.R

```diff
@@ -408,7 +407,7 @@ ms_aggregate = function(df, agg_selection, which_dataset,
         df = group_by(df, datetime)
     }
 
-    if(which_dataset %in% c('grab', 'pchem')){
+    if(which_dataset %in% c('chem', 'pchem')){
```

Comment:
which_dataset, depending how nested this is we could maybe use a more informative name. Cuz which dataset could also be which domain, watershed, or site. So this is really which_chem or which_datatype

Reply:
It's nested just one level down, but could be invoked deeper in the future. which_dataset is one of 'chem', 'q', 'p', 'pchem', so which_chem doesn't encompass the possibilities. And datatype to me refers to e.g. 'character' or 'double'. This is a perennial naming conundrum for me: the rare case when the truest, most specific identity of a variable seems to be simply "data".